mne/io/eeglab/eeglab.py

```diff
@@ -106,8 +106,8 @@ def _get_info(eeg, montage, eog=()):
     return info
 
 
-def read_raw_eeglab(input_fname, montage=None, preload=False, eog=(),
-                    verbose=None):
+def read_raw_eeglab(input_fname, montage=None, eog=(), misc=(), event_id=None,
```

Comment:
This is the order for the other readers. I assume nobody, or almost nobody, has used the function until now, considering it could not read events ...